### ISSUE TITLE
[Praise] Thanks for the PCB design and writing the library

### ISSUE THREAD

**Issue by depili** [hzeller/rpi-rgb-led-matrix#166]
Just wanted to thank you for your good work. Just finished soldering the active adapter today and everything is working fine :)

I noticed that the active adapter section doesn't currently have pictures of a as-built adapter of the current revision with rpi power in. If you want I can submit high res photos of my adapter to you.

![as built](https://c5.staticflickr.com/8/7517/27369109364_42c21253a0_h.jpg)
![as built](https://c5.staticflickr.com/8/7327/27880236012_7d6521e207_h.jpg)

Seeing how exited other people here at the Helsinki Hacklab are we might be building even more adapters quite soon :)

**Comment by depili** [comment]
Here are some high resolution photos of the built adapter.
![led_matrix_1](https://cloud.githubusercontent.com/assets/3681069/16994661/f78f6018-4eb0-11e6-87d0-7de4beae0f5d.jpg)
![led_matrix_2](https://cloud.githubusercontent.com/assets/3681069/16994662/f791f832-4eb0-11e6-84ad-cb69f09e16cd.jpg)

And I hereby place them in the public domain, so you are free to use them on the documentation if you so wish.

**Comment by Helbrax** [comment]
Agreed.  This is a great project.  One of the easiest libraries to use and extremely well documented actually.  I didn't use the board, but I designed a smaller one myself to run just one chain.

https://imgur.com/8j5dEKX

**Comment by hzeller** [comment]
Thanks for the pictures @depili ! I'll consider using them next time I update the README for the board.

**Comment by masto** [comment]
I just put mine together a couple of days ago as well - and it works, thanks so much for designing and sharing it! I agree, these pictures would be good as a gut check to make sure I wasn't putting anything on backwards. I assume it's either/or on the capacitors? This is mine (before I cleaned it).
![img_3412](https://cloud.githubusercontent.com/assets/645516/17351393/0e875c44-58fc-11e6-814e-9f563cd1a15e.jpg)

**Comment by depili** [comment]
Another project, a clock for AV production purposes, uses one 32x32 4mm pitch module and a 60 ws2812b led ring:
![Image of clock](https://c1.staticflickr.com/9/8452/29698912512_e86551f332_k.jpg)

**Comment by hzeller** [comment]
Very nice!

**Comment by marcmerlin** [comment]
With apologies to everyone, this project is doing a one time cleanup of 
all the bugs that have not been addressed and closed in the last 10 years.

If you filed an issue to ask for help, please do not re-open this one or a new
bug, but go here instead: https://rpi-rgb-led-matrix.discourse.group/

Please be mindful that this project does not have any full time maintainers
or anyone paid. You should not expect personal help for your own projects or
answers to programming questions but you can ask for help with panels that
aren't working, setup questions, or anything that may be a bug here:
https://rpi-rgb-led-matrix.discourse.group/
If you need programming help and are willing to pay someone for it, please
state so in your message, and someone may be interested.

If you did report an actual bug that is still not fixed today, please accept
our apologies for this being mass closed, you can absolutely send a followup
and re-open the issue, or open a new up to date one as applicable.
Note that we very much welcome help, so if you can help fix your bug, you will
be a superstar :)

If your bug was linked to a pull request that was not resolved, please update
the pull request and the issue, and we will do our best to review them. We do
thank you for your undertanding that we have very limited free time, so any
extra work you do along with your bug report or PR request will be very much
appreciated and improve the odds that they will be looked at.

If you want to help contribute as a python or C# or any other language bindings,
you are also a superstar, please state so in the issue and we'll do our best to
give you the access and reviews you need.

For anything related to PWM panel support, please make sure you read README and
then refer to https://github.com/hzeller/rpi-rgb-led-matrix/issues/466 . 
Please do not comment if you're haven't read everything and/or if you're just
asking when your panel is going to be supported. Those bugs are if you're 
able to contribute to the effort.

For rPI5: https://github.com/hzeller/rpi-rgb-led-matrix/issues/1603

For anything python, please link your bug to: https://github.com/hzeller/rpi-rgb-led-matrix/issues/1828
For anything C#, please link your bug to: https://github.com/hzeller/rpi-rgb-led-matrix/issues/1828
